test/integration/model/box_with_submodel/model.config

```
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<model>
+  <name>BoxWithSubmodel</name>
+  <version>1.0</version>
+  <sdf version="1.5">model.sdf</sdf>
```

nit: Should match the version in `model.sdf`
```suggestion
  <sdf version="1.6">model.sdf</sdf>
```